test/hashmaps/test_maps.fypp

```diff
@@ -249,6 +275,7 @@ module test_stdlib_open_maps
     integer, parameter        :: test_size = rand_size*4
     integer, parameter        :: test_16 = 2**4
     integer, parameter        :: test_256 = 2**8
+    integer, parameter        :: key_types = 2
```

Comment:
Please add a comment here along the lines of `! The tested key types are int8 and int32`

Reply:
Also added.